pandas/io/formats/format.py

```diff
@@ -1514,18 +1514,25 @@ def __init__(
 
     def _format_strings(self) -> List[str]:
         """ we by definition have DO NOT have a TZ """
-        values = self.values
-
-        if not isinstance(values, DatetimeIndex):
-            values = DatetimeIndex(values)
+        values = np.asarray(self.values)
+        flat_values = values.ravel() if len(values.shape) > 1 else values
```

Comment:
you can just do ravel unconditionally

Reply:
I was doing that at first, but in some cases `values` was an instance of `Index` and there is a deprecation warning for calling `ravel()`. I believe this was causing some tests to fail, but I can try again if you like.

Reply:
fair enough